tests/model_selection/test_keras.py

```diff
@@ -56,7 +58,12 @@ def test_keras(c, s, a, b):
     params = {"lr": loguniform(1e-3, 1e-1)}
```

Comment:
```suggestion
    params = {"lr": loguniform(1e-3, 1e-1), "random_state": [1]}
```

If we're fixing the random state, why not in the model too? This fixes the initialization (not the index ordering of the optimizer).